Sources/BraintreeApplePay/BTApplePayClient.swift

```diff
@@ -24,6 +24,9 @@ import BraintreeCore
 
     // MARK: - Public Methods
 
+    /// Creates a `PKPaymentRequest` with values from your Braintree Apple Pay configuration.
```

Comment:
We were missing docstring for all the public methods in here so added them back in